hexa/visualizations/admin.py

```diff
@@ -24,7 +24,7 @@ class IndexAdmin(admin.ModelAdmin):
     list_filter = ("content_type",)
     search_fields = ("label", "content")
 
-    def app_label(self, obj):
+    def app_id(self, obj):
```

Comment:
Why did you change `app_label` to `app_id` ?

Reply:
It was  a test in order to make the label clickable on the admin. But I'll remove it from now.